content/_data/changelogs/lts.yml

```diff
@@ -1429,6 +1429,54 @@
       message: >
         Prevent <code>NullPointerException</code> in <code>AbstractProject#checkout</code> when the agent disconnects during a build.
     # issue: 49634 wasn't notable enough originally, so skip for LTS as well
+- version: "2.107.3"
+  date: 2018-05-09
+  changes:
+    - type: security
+      message: Important security fixes.
+      references:
+        - url: https://jenkins.io/security/advisory/2018-05-09/
+          title: security advisory
+    - type: rfe
+      pull: 3403 # and 3404
+      references:
+        - issue: 50939
+        - issue: 50616
+      message:
+        Whitelist <code>java.util.EnumMap</code> and <code>org.jruby.RubyNil</code> for use in XStream (XML serialization) and Remoting (agent communication).
```

Comment:
Would it not be helpful to mention JEP-200 here?

Reply:
It would, but the both issues are not really widespread